### PR TITLE
NEW TEST COMMIT FOR ME

### PR DESCRIPTION
**Pull request by B0Step**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3807,3 +3807,27 @@ My  [Repository for Kottans Front-End](https://github.com/Chotogde/kottans-front
 also known as
 cat .git/config
 is a useful command by the way
+
+
+## Hello, my name is Bogdan and I study Git
```
Comment (suchov) [inline README.md:3812]: wow, so many Bohdans, do you really need them? )))

**Review by suchov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (B0Step) [reply README.md:3812]: Sorry, I'll fix it now)

**Review by B0Step — COMMENTED** [review]
Seems to have fixed it, if not please let me know. 
Son't forget i am a _beginner_))

File: README.md [hunk 2]
```diff
@@ -3843,6 +3845,31 @@ also known as
 cat .git/config
 is a useful command by the way
 
+<<<<<<< HEAD
```
Comment (suchov) [inline README.md:3848]: Now it also have this <<<<< >>>>> lines - they should be removed when you resolving conflicts ;)

**Review by suchov — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (B0Step) [reply README.md]: Hey,  chief dude, have I done everything?))

**Comment by suchov** [comment]
More or less, I'll help a bit ;)

**Review by suchov — APPROVED** [review]

**Comment by suchov** [comment]
Now you are done 🚀

**Pull request merged**